kong/pdk/client.lua

```diff
@@ -229,7 +232,7 @@ local function new(self)
   -- -- assuming `credential` and `consumer` have been set by some authentication code
   -- kong.client.authenticate(consumer, credentials)
   function _CLIENT.authenticate(consumer, credential)
```

Comment:
Does this function make sense in `response` phase since the proxy has already been done?